src/modules/Carousel.tsx

```diff
@@ -107,7 +106,7 @@ function CarouselSlide({ slide, offset }: CarouselSlideProps) {
         cssClass(CLASS_FLEX_CENTER),
       )}
       {...makeInertWhen(offscreen)}
-      onClick={handleBackdropClick}
+      // onClick={handleBackdropClick}
```

Comment:
No need to keep the unused code in comments. Please delete the code that is no longer in use.